src/Enum.php

```diff
@@ -83,9 +83,8 @@ public function getValue()
      * Returns the enum key (i.e. the constant name).
      *
      * @psalm-pure
-     * @return mixed
      */
-    public function getKey()
+    public function getKey(): string
```

Comment:
Unfortunately this is a BC break, as I stated here as well: https://github.com/myclabs/php-enum/pull/136#discussion_r575348768